docs/source/overview.rst

```diff
@@ -14,6 +14,7 @@ Features
 --------
 
 * No need to specify enquiries period (there is no *active.enquiries* status), since it overlaps with *active.tendering* period.
+* Lot exposition minimal period is 7 calendar days from tenderPeriod.startDate.
```

Comment:
The variant  "....... days (starting from tenderPeriod.startDate.)" would sound better.